core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/ExposedService.java

```diff
@@ -27,12 +27,12 @@
     SPARK_HISTORY_SERVER("Spark History Server", "SPARK_YARN_HISTORY_SERVER", "SPARKHISTORYUI", "/sparkhistory/", true, 18088, 18488, false, false),
     ZEPPELIN("Zeppelin", "ZEPPELIN_SERVER", "ZEPPELIN", "/zeppelin/", false, 8885, 8886, false, false),
     RANGER("Ranger", "RANGER_ADMIN", "RANGER", "/ranger/", true, 6080, 6182, false, true),
-    LIVY_SERVER("Livy Server", "LIVY_SERVER", "LIVYSERVER1", "/livy/ui", true, 8998, 8998, false, true),
+    LIVY_SERVER("Livy Server", "LIVY_SERVER", "LIVYSERVER1", "/livy/ui/", true, 8998, 8998, false, true),
     OOZIE_UI("Oozie Server", "OOZIE_SERVER", "OOZIE", "/oozie/", true, 11000, 11443, false, true),
     SOLR("Solr Server", "SOLR_SERVER", "SOLR", "/solr/", true, 8983, 8985, false, true),
     HBASE_UI("HBase UI", "MASTER", "HBASEUI", "/hbase/webui/master", true, 16010, 16010, false, false),
```

Comment:
Not sure if this should be `/hbase/webui/master/`

Reply:
It worked for me, but it worked also without the trailing /. I just adjusted it as the JIRa described.

Reply:
Oh, I commented on the wrong line. Yes, so I can change it here and then I don't need to change in the String.format section above.